azure/postgresql/terraform/modules/postgres/variables.tf

```diff
@@ -0,0 +1,112 @@
+### PostgreSQL server variables
+
+#Resource group name for potgresql server
+variable "postgres_resource_group_name" {
+  type = string
+  description = "Name of resource group to hold postgres"
+}
+#Resource group tags for potgresql server
+variable "postgres_resource_group_tags" {
+  type = map
+  description = "Tags to be added to the postgres resource group"
+}
+#Resource group location for potgresql server
+variable "postgres_resource_group_location" {
+  type = string
+  description = "The location of the resource group which will hold postgres"
+}
+#Name for postgres server
+variable "postgres_name" {
+  type = string
+  description = "Name for postgres server"
+}
+#Location of postgres server
+variable "postgres_location" {
+  type = string
+  description = "Location of the resource group"
+}
+#Version for postgres server
+variable "postgres_version" {
+  type = string
+  description = "Version of porstgres"
+}
+variable "postgres_tags" {
+  type = map
+}
+#Database storage in megabytes
+variable "storagemb" {
+  type = string
+  description = "Amount of storage for the database, in mb."
+  default = "5120"
+}
+#Number of days to retain backups
+variable "backup_retention_days" {
+  type = number
+  default = 7
+}
+#Enable geo redundency for Azure postgres
+#Enable geo redundancy for Azure postgres
+variable "geo_redundent_enabled" {
+  type = bool
+  description = "Geo redundent postgres enablement, currently supports only: true"
```

Comment:
I'm curious, why does this support only `true`? I noticed this below as well. Are these intended to be improved upon later- or why have a variable with only one supported value?